pkg/pinentry/cli/fallback.go

```diff
@@ -7,11 +7,13 @@ import (
 )
 
 // Client is pinentry CLI drop-in
-type Client struct{}
+type Client struct {
+	repeat bool
+}
 
 // New creates a new client
 func New() (*Client, error) {
-	return &Client{}, nil
+	return &Client{repeat: false}, nil
```

Comment:
It's not clear to me why we would want to disable repeat by default for new clients.
What's the motivation there?

Reply:
This module is being used as a drop-in replacement for pinentry, so the motivation is to replicate pinentry's behavior.